examples/gno.land/r/stefann/home/home.gno

```diff
@@ -0,0 +1,274 @@
+package home
+
+import (
+	"sort"
+	"std"
+
+	"gno.land/p/demo/ufmt"
+
+	"gno.land/r/stefann/config"
+)
+
+type City struct {
+	Name string
+	URL  string
+}
+
+type Sponsor struct {
+	Address std.Address
+	Amount  std.Coins
+}
+
+var (
+	pfp              string
+	cities           []City
+	currentCityIndex int
+	aboutMe          [2]string
+	jarLink          string
+	maxSponsors      int
+	sponsors         []Sponsor
+	totalDonated     std.Coins
+	totalDonations   int
+)
+
+func init() {
+	pfp = "https://i.ibb.co/Bc5YNCx/DSC-0095a.jpg"
+	cities = []City{
+		{Name: "Venice", URL: "https://i.ibb.co/1mcZ7b1/venice.jpg"},
+		{Name: "Tokyo", URL: "https://i.ibb.co/wNDJv3H/tokyo.jpg"},
+		{Name: "São Paulo", URL: "https://i.ibb.co/yWMq2Sn/sao-paulo.jpg"},
+		{Name: "Toronto", URL: "https://i.ibb.co/pb95HJB/toronto.jpg"},
+		{Name: "Bangkok", URL: "https://i.ibb.co/pQy3w2g/bangkok.jpg"},
+		{Name: "New York", URL: "https://i.ibb.co/6JWLm0h/new-york.jpg"},
+		{Name: "Paris", URL: "https://i.ibb.co/q9vf6Hs/paris.jpg"},
+		{Name: "Kandersteg", URL: "https://i.ibb.co/60DzywD/kandersteg.jpg"},
+		{Name: "Rothenburg", URL: "https://i.ibb.co/cr8d2rQ/rothenburg.jpg"},
+		{Name: "Capetown", URL: "https://i.ibb.co/bPGn0v3/capetown.jpg"},
+		{Name: "Sydney", URL: "https://i.ibb.co/TBNzqfy/sydney.jpg"},
+		{Name: "Oeschinen Lake", URL: "https://i.ibb.co/QJQwp2y/oeschinen-lake.jpg"},
+		{Name: "Barra Grande", URL: "https://i.ibb.co/z4RXKc1/barra-grande.jpg"},
+		{Name: "London", URL: "https://i.ibb.co/CPGtvgr/london.jpg"},
+	}
+	currentCityIndex = 0
+	jarLink = "https://TODO"
```

Comment:
The jarLink is not a leftover; it needs to be updated, using updateJarLink function, after the realm is deployed, as it's intended to link to the realm's own `Donate` function on Studio Connect. I added a comment to clarify its purpose and explain how it should be used (commit: aeb834048e930316533b6fe739748c24d21a57cc).

Reply:
Technically, you could predict the `Donate` URL if you knew which testnet you'd be using for the realm, but in my opinion, this would just add unnecessary complexity.

For example, the `jarLink` for the home realm on the portal loop would be:  
https://gno.studio/connect/view/gno.land/r/stefann/home?network=portal-loop#Donate

Reply:
Ah, got it, so it's a connect link to call a method 👍